packages/cli/src/lifecycles/scaffold.js

```diff
@@ -71,30 +71,25 @@ const writePageComponentsFromTemplate = async (compilation) => {
 
 };
 
-const writeListImportFile = async (compilation) => {
-  const importList = compilation.graph.map(file => {
-    return `import ${file.relativeExpectedPath};\n`;
-  });
-
-  // Create app directory so that app-template relative imports are correct
-  const appDir = path.join(compilation.context.scratchDir, 'app');
-  
-  await fs.ensureDir(appDir);
-  return await fs.writeFile(path.join(appDir, './list.js'), importList.join(''));
-};
-
 const writeRoutes = async(compilation) => {
   return new Promise(async (resolve, reject) => {
     try {
       let data = await fs.readFile(compilation.context.appTemplatePath, 'utf8');
 
       const routes = compilation.graph.map(file => {
-        return `<lit-route path="${file.route}" component="eve-${file.label}"></lit-route>\n\t\t\t\t`;
+        return `<lit-route
+          path="${file.route}"
+          component="eve-${file.label}"
+          .resolve="\${() => import(/* webpackChunkName: "${file.chunkName}" */ ${file.relativeExpectedPath})}"
+          ></lit-route>\n\t\t\t\t`;
```

Comment:
what's the `\n\t\t\t\t` for?  And foe the first `$` need to be escaped?

Reply:
just for code formatting, not necessary to function.

Reply:
Not sure I follow.  Formatting in the rendered HTML?   Not sure that has ever really been a concern?

Reply:
Ah, I see.  


<img width="1431" alt="Screen Shot 2020-06-03 at 8 49 40 PM" src="https://user-images.githubusercontent.com/895923/83702893-fa101400-a5db-11ea-9d7d-69584185cf0a.png">
It is the difference between this (with)


and this (without)
<img width="1616" alt="Screen Shot 2020-06-03 at 8 49 49 PM" src="https://user-images.githubusercontent.com/895923/83702883-f2506f80-a5db-11ea-8211-79d84bf027bf.png">

I don't think it's a big deal, and we can probably omit it.  I think if anything, we should / could use a html minfier on the final output just save even more bytes.  I'll handle removing the formatting and tracking html minification.